<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentFrame</class>
 <widget class="QFrame" name="StudentFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="lay_line1" stretch="1,2">
     <item>
      <layout class="QVBoxLayout" name="lay_login">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="login_lbl">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="login_edt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lay_fio">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="fio_lbl">
         <property name="text">
          <string>Fullname N. P.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fio_edt"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="lay_email">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="email_lbl">
       <property name="text">
        <string>e-mail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="email_edt"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="lay_comment">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="comment_lbl">
       <property name="text">
        <string>Comment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="comment_edt"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>login_edt</tabstop>
  <tabstop>fio_edt</tabstop>
  <tabstop>email_edt</tabstop>
  <tabstop>comment_edt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
